src/mercator-viewport.js

```diff
@@ -187,8 +189,8 @@ export default class MercatorViewport {
    *   Specifies a point on the sphere to project onto the map.
    * @return {Array} [x,y] coordinates.
    */
-  projectFlat([x, y], scale = this.scale) {
-    return this._projectFlat(...arguments);
+  projectFlat(xyz, scale = this.scale) {
+    return xyz;
```

Comment:
Not sure it matters, but what does returning xyz here mean? 

Should this assert? If just a dummy return value, maybe a comment?

Reply:
Yes, this is a dummy value. `projectFlat` is overridden by `PerspectiveMercatorViewport`.